vite.config.ts

```diff
@@ -15,4 +13,4 @@ export default defineConfig({
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
```

Comment:
The changes are minor and mostly involve removing unnecessary whitespace and comments. However, the removal of the trailing newline at the end of the file is not recommended as it's considered a best practice to include one.```javascript
})
```